### PR TITLE
Create midonet-manager ansible role

### PR DESCRIPTION
**Pull request by saptaktakalkar**

### CODE REVIEW

**Comment by saptaktakalkar** [comment]
@sibradzic Please review and merge.

File: handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Restart Midonet Manager
+  service:
+    name: midonet-manager
```
Comment (sibradzic) [inline handlers/main.yml]: We ain't restarting 'midonet-manager' there is no such service, its an apache WEB app, so Apache needs to get restarted. Meaning, this role also depends on apache, we should fail if apache or httpd packages aint there (although in theory it should work with any web server).

File: templates/client.js.j2 [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "api_namespace": "midonet-api",
+  "api_version": "{{ midonet_version }}",
+  "api_token": false,
+  "agent_config_api_namespace": "conf",
+  "analytics_ws_api_url": "wss://{{ midonet_api_hostname }}:8080/analytics",
+  "poll_enabled": true,
+  "login_animation_enabled": true,
+  "subscriptions_ws_api_url": "wss://{{ midonet_api_hostname }}:8007/subscription",
+  "fabric_ws_api_url": "wss://{{ midonet_api_hostname }}:8009/fabric"
+}
```
Comment (sibradzic) [inline templates/client.js.j2:15]: please change introduce a SSL switch in this template, defaulting to false. When SSL=false, all endpoints should start with http and ws, and when SSL=true all endpoints should start with https and wss

File: tests/test.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+
+- hosts: all
+  become: True
+  roles:
+    - role: ansible-midonet-repo
+      midonet_version: 'midonet-5.2'
+      mem_username: 'calsoft'
+      mem_password: '3adfF3cd'
```
Comment (sibradzic) [inline tests/test.yml]: DO NOT put valid credentials in here, ever. I had to revoke this password, affecting many people's work schedule. Put a dummy pwd instead.

**Review by sibradzic — CHANGES_REQUESTED** [review]

File: tests/test.yml (hunk 3 above)
Comment (saptaktakalkar) [reply tests/test.yml]: If we put dummy password then how would we able to test this role?

Comment (saptaktakalkar) [reply tests/test.yml]: CI will fail

File: templates/client.js.j2 (hunk 2 above)
Comment (saptaktakalkar) [reply templates/client.js.j2:15]: the default of midonet-cluster only accepts SSL-enabled connection, so we need to default to WSS and HTTPS

Comment (sibradzic) [reply templates/client.js.j2:15]: These defaults will also change ASAP, so please change it here as well.

File: tests/test.yml (hunk 3 above)
Comment (sibradzic) [reply tests/test.yml]: CI jobs will need to pass the valid credentials somehow, we can not expose the real credentials on a public github repository. Change it.

**Pull request merged**